### PR TITLE
docs: add roadmap in readme

### PR DESCRIPTION
**Pull request by aladdin-add**
refs: https://github.com/kataw/kataw/discussions/128

### CODE REVIEW

**Comment by KFlash** [comment]
LGTM. But move the Roadmap to end of the readme.

**Comment by KFlash** [comment]
I will fix v. 0.1, but I let you handle the bullet points for v. 0.2 👍

File: README.md [hunk 1]
```diff
@@ -13,6 +13,28 @@
 
 Kataw is a javascript toolchain with high focus on performance, and it's main goal is to unify functionality that has previously been separate tools.
 
+## 📌v0.1 roadmap
```
Comment (KFlash) [inline README.md:16]: Move to the bottom of the file

**Review by KFlash — CHANGES_REQUESTED** [review]
Move to bottom of the readme, not at the top.

**Comment by KFlash** [comment]
@aladdin-add friendly ping

**Comment by KFlash** [comment]
@aladdin-add friendly ping

**Comment by KFlash** [comment]
@aladdin-add I think the readme in this PR is kind of messed up ;)

**Comment by aladdin-add** [comment]
missing a blank line before the toc, should be addressed now. :)

**Comment by KFlash** [comment]
Shouldn't be added **after** the intro text? Makes it more readable.

And missing a section about printing

**Comment by aladdin-add** [comment]
> Shouldn't be added after the intro text? Makes it more readable.

yes, it should be. but the intro is a bit too long. it makes sense the toc be in the first page. thoughts?

**Comment by KFlash** [comment]
You either split up the intro or rewrite it and then merge.  Everything is fine.

**Comment by KFlash** [comment]
I still think you need to adjust the readme. Now at the top 'kataw' is used 3x. Should reduce to 1.

**Comment by KFlash** [comment]
I still think you need to adjust the readme. Now at the top 'kataw' is used 3x. Should reduce to 1.

**Review by KFlash — APPROVED** [review]

**Review by KFlash — APPROVED** [review]

**Pull request merged**